src/Raven.Server/Utils/ChangeVectorUtils.cs

```diff
@@ -216,10 +216,9 @@ public static string MergeVectors(string vectorAstring, string vectorBstring)
             if (string.IsNullOrEmpty(vectorBstring))
                 return vectorAstring;
 
-            if (_mergeVectorBuffer == null)
-                _mergeVectorBuffer = new EquatableList<ChangeVectorEntry>();
```

Comment:
Was there any reason why it was `EquatableList`?

Reply:
No, that is why I made the change.